web-wallet/src/lib/stores/stores.d.ts

```diff
@@ -63,10 +61,19 @@ type NetworkStoreServices = {
     options?: NetworkSyncerOptions
   ) => Promise<import("$lib/vendor/w3sper.js/src/mod").AddressSyncer>;
   getCurrentBlockHeight: () => Promise<bigint>;
+  init: () => Promise<void>;
 };
 
 type NetworkStore = Readable<NetworkStoreContent> & NetworkStoreServices;
 
+type NodeInfo = {
+  bootstrappingNodes: Array<string>;
+  chainId: number | null;
```

Comment:
Seems quite odd that a `number` can be `null`: please double check `w3sper`.

Reply:
If you're using w3sper, it normalize the null to a number. But, if you call the endpoint directly, without using w3sper, then it can be null.

So this definition will work in both scenarios.

Reply:
Yes, but we will use only w3sper and the additional `null` will force us to extra `null` checks and optional chainings.
Consider also that this type will go away when we release types for w3sper.

Reply:
We're using the null type in the explorer right now, so this copy was intended to be a copy and not a mutated copy.  However, if you feel strongly about the null checks, I can remove it.